### PR TITLE
QPPA-2737 Update 2019 IA Measures

### PR DESCRIPTION
**Pull request by shebryant**
This PR updates 2019 IA measures.

Tests pass and I used the validation tool to validate data.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "qpp-measures-data",
-  "version": "1.16.0",
+  "version": "1.16.1",
```
Comment (kalvinwang) [inline package.json]: Sorry you'll have to bump this to .2 now!

File: util/measures/2019/ia-measures.csv [hunk 2]
```diff
@@ -101,8 +101,8 @@ PSH Care Coordination,"Participation in a Perioperative Surgical Home (PSH) that
...
+Relationship-Centered Communication,"In order to receive credit for this activity, MIPS eligible clinicians must participate in a minimum of eight hours of training on relationship-centered care80 tenets such as making effective open-ended inquiries; eliciting patient stories and perspectives; listening and responding with empathy; using the ART (ask, respond, tell) communication technique to engage patients, and developing a shared care plan. The training may be conducted in formats such as, but not limited to: interactive simulations practicing the skills above, or didactic instructions on how to implement improvement action plans, monitor progress, and promote stability around improved clinician communication.",IA_CC_18,Care Coordination,medium,,,,boolean,2019,,medium,careCoordination,
+Use of certified EHR to capture patient reported outcomes,"use, etc.) or patient activation measures through use of certified EHR technology, containing this data in a separate queue for clinician recognition and review.",IA_BE_1,Beneficiary Engagement,medium,,,,boolean,2017,,medium,beneficiaryEngagement,
```
Comment (kalvinwang) [inline util/measures/2019/ia-measures.csv]: this new description kind of looks like a typo bc it starts lowercase and has an ending parenthesis... shrug

Comment (shebryant) [reply util/measures/2019/ia-measures.csv]: Thanks for catching this Kalvin! I checked with Sarah and she's uploaded an updated file. I'll make the necessary changes.

File: measures/2019/measures-data.json [hunk 3]
```diff
@@ -9890,7 +9890,7 @@
     "metricType": "boolean",
     "firstPerformanceYear": 2017,
     "lastPerformanceYear": null,
-    "weight": "high",
+    "weight": "medium",
```
Comment (kalvinwang) [inline measures/2019/measures-data.json]: since the other change was unintended maybe it's worth checking on this one too? cc @sarahnw

**Pull request merged**